schemas/top-api.yml

```diff
@@ -2728,8 +2732,10 @@ components:
         code:
           type: string
           example: 3137-7
-        scope:
-          $ref: '#/components/schemas/CodeScope'
+        children:
```

Comment:
I need to check the implementations in the other components before deciding on this change. Originally it was planned for the backend to collect sub codes. By replacing `scope` with `children`, the work now has to be done by the frontend, resulting in a potentially large number of sub codes.

Reply:
The codes **are** collected by / stored in the backend. They need to be presented in the frontend, though, as it was planned to give users the ability to manually edit the subtrees (i.e., delete unneeded branches/leaves).